pilot/AccountTableViewController.swift

```diff
@@ -42,14 +42,24 @@ class AccountTableViewController: UITableViewController {
             if indexPath.row == 0 {
 
                 if let type = self.platform?.type {
-                    PlatformManager.sharedInstance.disconnectPlatform(type: type, onSuccess: {
+                    let actionSheet = UIAlertController(title: "Disconnect" + type.rawValue,
```

Comment:
I think the title will be missing a space here. As in, it'll say "DisconnectFacebook" instead of "Disconnect Facebook"